File: search/WeexSearch/app/src/main/java/com/tao/weexsearch/activity/MainActivity.java
package com.tao.weexsearch.activity;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;

import com.tao.weexsearch.R;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void start_ac_loadjs(View view){
        startActivity(new Intent(MainActivity.this,RecommendComponentActivity.class));
    }
}

File: foregroudService/app/src/main/java/com/tao/foregroudservice/MainActivity.kt
package com.tao.foregroudservice

import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        bt_notification.setOnClickListener { view -> run{
            var intent = Intent("com.tao.my.service")
            startService(intent)
        } }
    }
}
